manifests/f/FastCopy/FastCopy/3.92/FastCopy.FastCopy.yaml

```diff
@@ -1,7 +1,7 @@
 PackageIdentifier: FastCopy.FastCopy
 PackageVersion: 3.92
 PackageName: FastCopy
-Publisher: FastCopy
+Publisher: H.Shirouzu & FastCopy Lab, LLC
```

Comment:
Hey there, `Publisher` needs to match the one you find in the control panel.

![image](https://user-images.githubusercontent.com/52826317/124345706-c63a6000-dbda-11eb-9201-23ca0f389397.png)


```suggestion
Publisher: H.Shirouzu
```

Reply:
Oh... I will modify the control panels description to "H.Shirouzu and FastCopy Lab, LLC." in the next release.

But now, Publisher is FastCopy(this commit is by yao-msft), and your suggestion is  "H.Shirouzu".
If this suggestion is commited,  the file should be moved "s/Shirouzu/FastCopy" and packege-id should be "Shirouzu.FastCopy" ?

May I cancel this PR and send a PR that contains "Publisher: H.Shirouzu & FastCopyLab; PackageIdentifier: FastCopyLab.FastCopy" at the next release that version's controlpanel-desc is "H.Shirouzu and FastCopy Lab, LLC.".

Reply:
> Oh... I will modify the control panels description to "H.Shirouzu and FastCopy Lab, LLC." in the next release.

The Publisher/PackageName/Versions needs to match whats in Windows Add and Remove Programs (ARP)

> But now, Publisher is FastCopy(this commit is by yao-msft), and your suggestion is "H.Shirouzu".
If this suggestion is commited, the file should be moved "s/Shirouzu/FastCopy" and packege-id should be "Shirouzu.FastCopy" ?

The manifest metadata was created by "auanasgheps" 13 months ago when we didn't have these guidelines.
The Publisher and PackageName does not need to match the PackageIdentifier, I think FastCopy is better since its a Company Name.

Reply:
The publisher for this version should be `H.Shirouzu` so it can catch all the old versions which have the Publisher H.Shirouzu.
Since you fixed the Publisher in the new version ONLY that version and newer will be registerd in WinGet List/WinGet Upgrade.